include/SampleFrame.h

```diff
@@ -0,0 +1,234 @@
+/*
+ * SampleFrame.h - Representation of a stereo sample
+ *
+ * Copyright (c) 2004-2009 Tobias Doerffel <tobydox/at/users.sourceforge.net>
+ * Copyright (c) 2024- Michael Gregorius
+ *
+ * This file is part of LMMS - https://lmms.io
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program (see COPYING); if not, write to the
+ * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301 USA.
+ *
+ */
+
+#pragma once
+
+#ifndef SAMPLEFRAME_H
+#define SAMPLEFRAME_H
```

Comment:
I don't believe there's any advantage to additionally including `#pragma once` if a standard macro-based include guard is already present. As for the latter, we now prefix the namespace to it too (i.e. `LMMS_SAMPLEFRAME_H`).

Reply:
Adjusted with commit c8f3a007a77.